js/simple/package.json

```diff
@@ -16,6 +16,7 @@
   "dependencies": {
   },
   "devDependencies": {
-    "azure-iot-gateway": "~1.1.4"
+    "azure-iot-gateway": "~1.1.4",
+    "azure-iot-device-amqp": "~1.1.17"
```

Comment:
we might want to check with the owner to see which way to go, since MQTT has been the direction and AMQP has been deprecated since the past 6 months:

1. continue using AMQP
2. move to MQTT

Reply:
I will start a conversation with the IoT Edge SDK team to see what direction they want to go, keep it as it is or change the sample to use the MQTT protocol instead. If it is the latter I will update the packages.json to reflect the change.